docs/usage.md

```diff
@@ -52,12 +50,105 @@ TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
+## Example workflow
+
+An example workflow for RNA-seq data from an experiment can look like this:
+
+1. Run the pipeline with `--qc_only` with default params to check the quality of your reads.
+
+```bash
+nextflow run nf-core/denovotranscript \
+  --input samplesheet.csv \
+  --outdir <OUTDIR> \
+  -profile docker \
+  --qc_only
+```
+
+2. Run the pipeline with `qc_only` and any custom params that you have decided to use based on your data.
```

Comment:
```suggestion
2. Run the pipeline with `qc_only` and any custom parameters that you have decided to use based on your data. Use `resume` to avoid unnecessarily rerunning unchanged steps
```